Book-A-Desk.Api/BookingsHttpHandler.fs

```diff
@@ -24,7 +25,58 @@ type BookingsHttpHandler =
     }
 
 module BookingsHttpHandler =
-    let initialize (provideEventStore : IAmazonDynamoDB -> DynamoDbEventStore) reservationCommandsFactory (notifySuccess: Models.Booking-> Async<bool>) =
+    let initialize
+        (provideEventStore : IAmazonDynamoDB -> DynamoDbEventStore)
+        reservationCommandsFactory
+        (notifySuccess: Models.Booking-> Async<bool>)
+        (errorHandler: BookADeskErrorHandler) =
+
+        let notifyBooking output booking next context =
```

Comment:
Small suggestion:
BookingsHttpHandler.initialize is starting to become quite a big function. I suggest you extract notifyBooking and handleCommand from initialize (as private functions in BookingsHttpHandler)

Reply:
I've done a bit of refactor. Now, that functions are private.